src/output.cpp

```diff
@@ -1383,8 +1458,8 @@ void draw_subtab(WINDOW *w, int iOffsetX, std::string sText, bool bSelected, boo
 }
 
 void draw_scrollbar(WINDOW *window, const int iCurrentLine, const int iContentHeight,
-                    const int iNumEntries, const int iOffsetY, const int iOffsetX,
-                    nc_color bar_color)
+                    int iNumEntries, const int iOffsetY, const int iOffsetX,
+                    nc_color bar_color, bool bRefresh, int iFakeEntries)
```

Comment:
If there are 10 items/lines in a list but only 5 rows to display them, you can scroll 10 times with the current behaviour, which is ok because you want to be able to select every single line.

For displaying only text, without selecting anything, I only want to scroll 5 times. Otherwise you would get blank lines at the bottom like you said. But as it was, if numentries (5) is less or equal to display height(5) it won't let me scroll. Couldn't come up with a better solution.